doc/manual/stacktraces.rst

```diff
@@ -0,0 +1,213 @@
+.. _man-stacktraces:
+
+.. currentmodule:: Base
+
+Stack Traces
+============
+
+The :mod:`StackTraces` module provides simple stack traces that are both human readable and easy to use programmatically.
+
+Viewing a stack trace
+---------------------
+
+The primary function used to obtain a stack trace is :func:`stacktrace`::
+
+    julia> using StackTraces
+
+    julia> stacktrace()
+    2-element Array{StackTraces.StackFrame,1}:
```

Comment:
I think there's supposed to be some filtering out of those last few lines, but it may have been broken since the big inlined-line-numbers change a few months back.